Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -1305,6 +1212,29 @@ def get_exclusion(self, limit, tenant_id=None, filter=None):
         reply = res.get("reply")
         return reply[:limit]
 
+    def add_or_remove_tag_endpoint(self, endpoint_ids, tag, filters, add_tag=True):
```

Comment:
Matter of style:
I probably wouldve written it like this:
```
def add_endpoint_tag(self,  endpoint_ids, tag, filters)
    return call_tag_endpoint(endpoint_ids, tag, filters, '/tags/agents/assign/',)

def remove_endpoint_tag(self,  endpoint_ids, tag, filters)
    return call_tag_endpoint(endpoint_ids, tag, filters, '/tags/agents/remove',)

def call_tag_endpoint(self, endpoint_ids, tag, filters, endpoint)
    pass
```

I think it makes it a bit more clear at the callsite.

Reply:
Agreed, will change